client/src/protocol/types.ts

```diff
@@ -51,7 +51,7 @@ export interface ProofViewNotification {
     messages: CoqMessage[];
 }
 
-export interface UpdateHightlightsNotification {
+export interface UpdateHighlightsNotification {
     uri: Uri; 
     parsedRange: Range[];
```

Comment:
```suggestion
    processingRange: Range[];
```
After checking this is the current semantic. Thanks for the changes :-)